releasenotes/notes/set-penatly-to-one-zero-objective-116b50b47ad41051.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Set the lagrange multiplier to one if the maximum absolute bias is zero
```

Comment:
can you add `... in the ``cqm_to_bqm()`` function.` to the end of this?

Reply:
Also if you could capitalize: "the Lagrange multiplier"